src/main/java/vectorwing/farmersdelight/utils/tags/ForgeTags.java

```diff
@@ -40,6 +40,8 @@
 	public static final ITag.INamedTag<Item> MILK_BUCKET = forgeItemTag("milk/milk");
 	public static final ITag.INamedTag<Item> MILK_BOTTLE = forgeItemTag("milk/milk_bottle");
 
+	public static final ITag.INamedTag<Item> MUSHROOMS = forgeItemTag("mushrooms");
```

Comment:
Just a heads-up: `forge:mushrooms` is already an official Forge Tag, and it's working on the current version.; hence why I didn't fill it 😄

Reply:
I did try to use it, but runData failed because I suppose it wasn't made yet or something

Reply:
Hmm, that's weird... Usually I refer to official Forge tags when datagenning some recipes (such as Shears-based cutting, or the current Cabbage Rolls recipe). Perhaps it bugs out when using them to fill new tags? Not sure if I ever had to do this in specific yet.

Reply:
After doing some research I've discovered this is an issue solved by Forge version 35.1.3 but since FD is not on that internally it does not work without crashing

Reply:
Good catch! I haven't updated it for a while, mostly because I wanted to continue supporting 1.16.3. Do you know if Forge 35-26 is backwards-compatible on build? I'm not very savvy on that front.

Reply:
Sorry, forgot to respond. Not sure. I assume this would work fine since there shouldn't be anything related to the data generators when the mod is built but you never know. I'll take a look later

Reply:
Ok, after testing it out: the mod builds fine if you aren't using 35.1.3, but to runData you'll need to have the workspace in 35.1.3. So you don't have to force a new version. (although, I don't think forcing a later version would hurt since most people are on *at least* 1.16.4, and should really be on 1.16.5.